Orange/data/variable.py

```diff
@@ -632,21 +632,16 @@ class DiscreteVariable(Variable):
     .. attribute:: values
 
         A list of variable's values.
-
-    .. attribute:: ordered
-
-        Some algorithms (and, in particular, visualizations) may
-        sometime reorder the values of the variable, e.g. alphabetically.
-        This flag hints that the given order of values is "natural"
-        (e.g. "small", "middle", "large") and should not be changed.
     """
 
     TYPE_HEADERS = ('discrete', 'd', 'categorical')
 
     presorted_values = []
 
-    def __init__(self, name="", values=(), ordered=False, compute_value=None,
-                 *, sparse=False):
+    def __init__(
+            self, name="", values=(), compute_value=None, sparse=False,
+            **kwargs
+    ):
```

Comment:
This is a bit dangerous: any call that passes `compute_value` as non-keyword argument will (possibly silently) fail. What about keeping `ordered` as an argument, but changing its default to `None` and the test `if "ordered" in kwargs:` to `if ordered is not None`?

Though: this is unlikely and, besides, such calls won't be fixed until the argument is actually removed and the calls will actually fail. We ignore warnings, including those about deprecations.

Reply:
You are right. Didn't think about this possibility. This solution was to avoid problems with __repr__ but now I fixed it differently.